overlord/snapstate/snapstate_test.go

```diff
@@ -8177,9 +8232,12 @@ func (s *snapmgrTestSuite) TestSnapManagerRefreshSchedule(c *C) {
 		{"", snapstate.DefaultRefreshSchedule},
 		{"invalid schedule", snapstate.DefaultRefreshSchedule},
 		{"8:00-12:00", "8:00-12:00"},
+		// using legacy configuration, this is parsed by the legacy schedule parser
+		{"0:00~24:00/24", snapstate.DefaultRefreshSchedule},
```

Comment:
I think this tests that the legacy parser does not parse new style timer configuration? If so, maybe make the comment a little more clear: something like `// using the legacy configuration option with a new-style refresh.timer string is rejected (i.e. the legacy parser is used for the parsing)` ?